### PR TITLE
#5143 spacing issue explore

### LINKED ISSUE #5143: Explore feedback leftovers
this looks to be almost finished, I will make maybe follow for these small changes? 

spacing is still bit of

The menu bar and the full width divider should have space 40px, applies to collection as well
image

cards and sidebar should have 32px 
image

_Originally posted by @exezbcz in https://github.com/kodadot/nft-gallery/issues/4906#issuecomment-1450503453_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5143
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-12 at 3 09 24 PM

### CODE REVIEW

**Review by daiagi — APPROVED** [review]

**Comment by exezbcz** [comment]
image
this to 40px please

File: layouts/explore-layout.vue [hunk 1]
```diff
@@ -77,4 +77,8 @@ hr {
     padding: 0 1rem;
   }
 }
+
+.explore-header {
+  padding-bottom: 40px;
+}
```
Comment (vikiival) [inline layouts/explore-layout.vue:83]: pls do not use px

**Review by vikiival — CHANGES_REQUESTED** [review]
as @exezbcz & I said Pls

**Comment by vikiival** [comment]
+ please resolve conflicts

File: components/explore/SidebarFilter.vue [hunk 2]
```diff
@@ -28,5 +28,6 @@ const open = computed(() => preferencesStore.getsidebarFilterCollapse)
   position: sticky;
   top: 84px;
   height: calc(100vh - 84px);
+  margin-right: 32px;
```
Comment (vikiival) [inline components/explore/SidebarFilter.vue:31]:
```suggestion
  margin-right: 5rem;
```

**Review by vikiival — APPROVED** [review]

**Pull request merged**